README.md

```diff
@@ -29,34 +29,28 @@
 
     We now have an AKS cluster named `bikesharingcluster` in resource group `bikesharinggroup` in the `eastus2` region with Azure Dev Spaces enabled.
 
-4. Select the `dev` parent space:
```

Comment:
@Yamini > Following your comment on Teams, I had a look at this step and actually it shouldn't even be needed: the step 3 is already taking care of creating and selecting a "dev" space. So I just removed step 4.